router/users-router.js

```diff
@@ -0,0 +1,109 @@
+const router = require('express').Router()
+const bcryptjs = require('bcryptjs');
+const session = require('express-session');
+const knexSessionStore = require('connect-session-knex')(session);
+
+const db = require('../data/knexConfig');
+const Users = require('../users/usersModel');
+
+// SESSION TO KEEP COOKIES
+const sessionSecret = 'this is a secret, keep it to yourself!'
+const sessionConfig = {
+    name: 'panther',
+    secret: sessionSecret,
+    cookie: {
+        maxAge: 1000 * 60 * 15,
```

Comment:
For a sanity check, make this and the session store interval to last longer during build week.